include/bluetooth/gap.h

```diff
@@ -163,6 +163,8 @@ enum {
 /** Maximum Periodic Advertising Interval (N * 1.25 ms) */
 #define BT_GAP_PER_ADV_MAX_INTERVAL             0xFFFF
 
+/** Convert advertising interval (N * 1.25 ms) to milliseconds */
+#define BT_GAP_ADV_INTERVAL_TO_MS(interval) ((interval) * 5 / 4) /* 5 / 4 represents 1.25 ms unit */
```

Comment:
```suggestion
/** Convert periodic advertising interval (N * 1.25 ms) to milliseconds */
#define BT_GAP_PER_ADV_INTERVAL_TO_MS(interval) ((interval) * 5 / 4) /* 5 / 4 represents 1.25 ms unit */
```

"Regular" ADV interval is slots of 0.625ms, whereas periodic is 1.25ms.

Reply:
Thanks! Good catch, changed.